src/introduction-to-unified.md

```diff
@@ -0,0 +1,215 @@
+## Introduction to unified
+
+* * *
+
+After reading this guide you will:
+
+*   Understand what unified does.
+*   Get a taste of the ecosystem.
+*   Know how it can be used.
+*   Know what parts (processors) you need for your (future) use case.
+*   Have a list of resources to continue learning or get started.
+
+* * *
+
+![Example of using unified](./image/unified-overview.png)
+
+**unified is a friendly interface backed by an ecosystem of plugins built for
+creating and manipulating content**. unified does this by taking markdown, HTML,
+or plain text prose, turning it into structured data, and making it available
+to over 100 plugins.  Tasks like text analysis, preprocessing, spellchecking,
+linting, and more can all be done through compatible tools, and even chained
+together.
+
+This and more is possible thanks to unified’s plugin pipeline, which lets you
+typically write one line of code to chain a feature into this process.
+It’s also possible to stitch together content from different sources
+and output it as a single source.
+
+**Bottom line: with unified, you don’t manually handle syntax or parsing.**
+
+However, the heavy lifting isn’t being done by unified itself, it’s a rather
+small module that acts as an interface to unify the handling of different
+content formats.  What makes unified powerful are the plugins that can be 
+chained to the handler of a content format.  A module that handles a certain
```

Comment:
The sentence `What makes unified powerful are the plugins that can be chained to the handler of a content format.` is re-iterating the previous paragraph

Reply:
Also maybe define `content format` on its first use, such as with `(such as Markdown, HTML, or prose)`